### ISSUE TITLE
Error: _react3.default.createClass is not a function

### ISSUE THREAD

**Issue by snowzurfer** [viromedia/viro#82]
Hi @VikAdvani , when would the next release be? I have started using react-native recently, and have tried to generate a MWE but because of the changes to react-native and the problem above it does not work. Here is my code:

### App.js
```
import React, { Component } from 'react';
import { StyleSheet } from 'react-native';
import {
  ViroARScene,
  ViroARSceneNavigator,
  ViroText,
} from 'react-viro';

export class ARScene extends Component {
  constructor(props) {
    super(props);
    this.state = {text: 'Initializing ProvAR...'};
  }

  render() {
    return (
      <ViroARScene onTrackingInitialized={()=>{
        this.setState({text: 'Hello World'});
      }}>
        <ViroText text={this.state.text} scale={[.5, .5, .5]}
          position={[0, 0, -1]} style={styles.textStyle} />
      </ViroARScene>
    );
  }
}

export default class App extends Component {
  render() {
    return (
      <ViroARSceneNavigator apiKey='MY_KEY_HERE'
        initialScene={{scene: ARScene}} />
    );
  }
}
```
### index.js
```
import { AppRegistry } from 'react-native';
import App from './App';

AppRegistry.registerComponent('provar', () => App);
```

The error I get, apart from the warnings as shown above, is:
![img](https://i.imgur.com/8DPJPmJ.png)

**Comment by VikAdvani** [comment]
Hi @snowzurfer, we plan for a release in mid november that will support React Native 0.49 which would remove the error you are getting. For now you can try downgrading your React Native version to 0.47 to get a MWE working. Let me know if you have any more questions!

**Comment by dthian** [comment]
Hi @snowzurfer , this issue should now be fixed in the latest release of Viro 2.1. 
To update, you can view the instructions [https://docs.viromedia.com/docs/releases](here). 
Please give it a shot and let us know if you encounter any issues. Thanks!

**Comment by achuvm** [comment]
Closing due to inactivity.

If you still have issues, feel free to re-open this issue or create a new one.

The latest releases are here:
React Viro - https://docs.viromedia.com/docs/releases
ViroCore - https://virocore.viromedia.com/docs/releases

Thanks for using Viro!
